docs/admin/upgrading_to_1.7.0.md

```diff
@@ -0,0 +1,23 @@
+# Upgrading to 1.7.0
+
+This note describes the necessary steps to upgrade to Stellio 1.7.0
+
+## Contact entity service
+
+Stellio now allows to send periodic notifications, for this it's necessary to add new variables to contact the entity service in the docker-compose. 
+In subscription-service, please add the following variables: 
+
+```
+# Client registration used to get entities from entity-service
+spring.security.oauth2.client.registration.keycloak.authorization-grant-type=client_credentials
+spring.security.oauth2.client.registration.keycloak.client-id=client-id
+spring.security.oauth2.client.registration.keycloak.client-secret=client-secret
+spring.security.oauth2.client.provider.keycloak.token-uri=https://my.sso/token
+application.entity.service-url=http://localhost:8082
+```
+
+If you are using Stellio without authentication, you only need to add the following line:
+
+```
+application.entity.service-url=http://localhost:8082
```

Comment:
btw, for people using the default docker compose in Stellio repository, we should set a default value of http://entity-service:8082

Reply:
It's the name of the service or the name of container that we need to put?

Reply:
of the service